config.json

```diff
@@ -48,76 +53,169 @@
 			"name" : "Enable @SQLDESCRIPTIVE action tag",
 			"type" : "checkbox"
 		},
+		{
+			"key" : "hdr-data-qual",
+			"name" : "<b>Data quality tweaks</b>",
+			"type" : "descriptive"
+		},
+		{
+			"key" : "data-res-workflow",
+			"name" : "Enable the Data Resolution Workflow on new projects",
+			"type" : "checkbox"
+		},
+		{
+			"key" : "missing-data-codes",
+			"name" : "Default missing data codes",
+			"type" : "textarea"
```

Comment:
Are these missing data codes added by admin user when configuring the module?  could we default them?

Reply:
Yes, the default missing data codes for new projects are specified in the module system settings.
I don't see any value in defaulting this setting to anything when installing the module as sensible default missing data codes could vary significantly between institutions based on sponsor requirements etc.